atd-vzv/src/views/summary/CrashesByYear.js

```diff
@@ -113,7 +113,7 @@ const CrashesByYear = () => {
           <hr />
         </Col>
       </Row>
-      {avgData.length > 0 && currentYearData.length > 0 ? (
+      {avgData && currentYearData ? (
```

Comment:
allows the loading spinner to stop if we just don't have any data for the current year